src/main/java/vazkii/botania/common/Botania.java

```diff
@@ -30,6 +30,7 @@
 import cpw.mods.fml.common.event.FMLServerAboutToStartEvent;
 import cpw.mods.fml.common.event.FMLServerStartingEvent;
 import cpw.mods.fml.common.event.FMLServerStoppingEvent;
+import vazkii.botania.common.network.PacketHandler;
```

Comment:
Please sort the imports (here and elsewhere)

Reply:
Have I done a good job ordering them now? I think I did it, might have missed somewhere still tho.

Reply:
It's certainly a lot better now. Remember, modern IDEs can do all the work for you with a single key combination (e.g. Ctrl+Alt+O on IntelliJ)